patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -268,24 +263,20 @@ function closeMenu() {
   menuButton.setAttribute("aria-pressed", "false");
 }
 
-function closeMenuJumpToSearch() {
-  commonCloseMenuTasks();
-  jumpToSearchButton.setAttribute("aria-pressed", "false");
-}
-
 function commonCloseMenuTasks() {
   body.classList.remove("show-menu");
 
   if (width < 841) {
     menuButton.removeAttribute("style");
   }
 
-  menuButtonText[0].textContent = "Menu";
```

Comment:
Removed these lines.  The label text change will be handled via css which is done in DP-19337.